### ISSUE TITLE
TypeError: Cannot read property 'confirmationBlocks' of undefined

### ISSUE THREAD

**Issue by jorisbontje** [nexusdev/dapple#283]
with dapple 0.7.2 when deploying to the 'live' environment that didn't already exist in my `dappfile` before.

```
$ dapple run scripts/deploy.ds -e live
/Users/mids/.node/lib/node_modules/dapple/cmd/main.js:260
  let confirmationBlocks = workspace.dappfile.environments[env].confirmationBlocks;
                                                               ^

TypeError: Cannot read property 'confirmationBlocks' of undefined
    at Object.<anonymous> (/Users/mids/.node/lib/node_modules/dapple/cmd/main.js:260:64)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
name: maker-otc
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:141:18)
    at node.js:933:3
```

**Comment by jorisbontje** [comment]
Any update on this?

**Comment by mhhf** [comment]
Sry for the delay, was busy with the 0.8 release, where this shouldn't be a problem. However, I fixed this for 0.7.2 as well, the new master branch contains this fix and will be available with 0.7.3 via npm
